### PR TITLE
Deployment script fixes

### PR DESCRIPTION
**Pull request by justinjmoses**
This fixes a few issues with the deployment script:

1. When not deploying a new `ExchangeRates`, this will no longer try to run `setInversePricing` unless the inverse parameters have changed;
2. `FeePool.targetThreshold` is checked to be `1%` as per https://sips.synthetix.io/sccp/sccp-6
3. Adds the useful `dryRun` feature to deploys

### CODE REVIEW

File: publish/deployed/kovan/synths.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
 		"category": "forex",
 		"sign": "$",
 		"desc": "Australian Dollars",
-		"aggregator": "0xfd4767f2136f277ddd36b563504797c46a2eff3b"
+		"aggregator": "0xFD4767F2136F277ddd36b563504797C46a2eFF3B"
```
Comment (justinjmoses) [inline publish/deployed/kovan/synths.json]: Drive-by fix - this address was not correctly cased here.

File: publish/src/commands/deploy.js [hunk 2]
```diff
@@ -354,11 +356,13 @@ const deploy = async ({
 			bytecode: compiled[source].evm.bytecode.object,
 			abi: compiled[source].abi,
 		};
-		fs.writeFileSync(deploymentFile, stringify(deployment));
+		if (!dryRun) {
```
Comment (justinjmoses) [inline publish/src/commands/deploy.js]: Doesn't mutate the `config.json` or `deployment.json` files during a dry run

File: publish/src/commands/deploy.js [hunk 3]
```diff
@@ -512,6 +517,19 @@ const deploy = async ({
 		});
 	}
 
+	if (feePool) {
+		// Set FeePool.targetThreshold to 1%
```
Comment (hav-noms) [inline publish/src/commands/deploy.js]: This is a nice protection. However I'm going to change the default to 1% from 10%.
